app/components/vector_icon.js

```diff
@@ -75,14 +66,6 @@ export default class VectorIcon extends PureComponent {
                     size={size}
                 />
             );
-        case 'mattermost':
```

Comment:
What is the purpose of removing this?

Reply:
this is used to convert custom SVG icons into font icons using ico-moon(a third party library), this requires a JSON file inside the asset folder containing custom icons but these custom icons are not used anywhere also this JSON file is missing in the required folder which gives import error.

This is generally a setup part that is not used anywhere so commented it as it is giving an error of missing JSON file that never existed

Reply:
This file is really old and I didn't get any error running the app till now. What error are you getting?

Reply:
the component of this file was not used anywhere before now, so it didn't produce any error